framework/src/Volo.Abp.EntityFrameworkCore/Volo/Abp/Uow/EntityFrameworkCore/UnitOfWorkDbContextProvider.cs

```diff
@@ -60,6 +61,8 @@ private TDbContext CreateDbContext(IUnitOfWork unitOfWork, string connectionStri
                     dbContext.Database.SetCommandTimeout(unitOfWork.Options.Timeout.Value.TotalSeconds.To<int>());
                 }
 
+                dbContext.ChangeTracker.CascadeDeleteTiming = CascadeTiming.OnSaveChanges;
+                dbContext.ChangeTracker.DeleteOrphansTiming = CascadeTiming.OnSaveChanges;
```

Comment:
Can we make this inside the constructor of the `AbpDbContext`? In this way, any dbcontext can change easy for itself.
What do you think?

Reply:
It is safe to use `ChangeTracker `in the constructor, I will update the PR.